sdk/ai/azure-ai-projects/README.md

```diff
@@ -19,52 +19,53 @@ For example, get the inference endpoint URL and credentials associated with your
 
 ## Table of contents
 
-- [Getting started](#getting-started)
-  - [Prerequisite](#prerequisite)
-  - [Install the package](#install-the-package)
-- [Key concepts](#key-concepts)
-  - [Create and authenticate the client](#create-and-authenticate-the-client)
-- [Examples](#examples)
-  - [Enumerate connections](#enumerate-connections)
-    - [Get properties of all connections](#get-properties-of-all-connections)
-    - [Get properties of all connections of a particular type](#get-properties-of-all-connections-of-a-particular-type)
-    - [Get properties of a default connection](#get-properties-of-a-default-connection)
-    - [Get properties of a connection by its connection name](#get-properties-of-a-connection-by-its-connection-name)
-  - [Get an authenticated ChatCompletionsClient](#get-an-authenticated-chatcompletionsclient)
-  - [Get an authenticated AzureOpenAI client](#get-an-authenticated-azureopenai-client)
-  - [Agents (Private Preview)](#agents-private-preview)
-    - [Create an Agent](#create-agent) with:
-      - [File Search](#create-agent-with-file-search)
-      - [Code interpreter](#create-agent-with-code-interpreter)
-      - [Bing grounding](#create-agent-with-bing-grounding)
-      - [Azure AI Search](#create-agent-with-azure-ai-search)
-      - [Function call](#create-agent-with-function-call)
-    - [Create thread](#create-thread) with
-      - [Tool resource](#create-thread-with-tool-resource)
-    - [Create message](#create-message) with:
-      - [File search attachment](#create-message-with-file-search-attachment)
-      - [Code interpreter attachment](#create-message-with-code-interpreter-attachment)
-    - [Execute Run, Run_and_Process, or Stream](#create-run-run_and_process-or-stream)
-    - [Retrieve message](#retrieve-message)
-    - [Retrieve file](#retrieve-file)
-    - [Tear down by deleting resource](#teardown)
-    - [Tracing](#tracing)
-  - [Evaluation](#evaluation)
-    - [Evaluator](#evaluator)
-    - [Run Evaluation in the cloud](#run-evaluation-in-the-cloud)
-      - [Evaluators](#evaluators)
-      - [Data to be evaluated](#data-to-be-evaluated)
-        - [[Optional] Azure OpenAI Model](#optional-azure-openai-model)
+- [Azure AI Projects client library for Python](#azure-ai-projects-client-library-for-python)
+  - [Table of contents](#table-of-contents)
```

Comment:
Please remove the top level item `Azure AI Projects client library for Python` and the one after it `Table of contents`, as TOC never point to the start of the article and never point to itself. They always start from the next section following the TOC.

Reply:
Done